sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { AtomResourceSerializerBase, ResourceSerializer } from "@azure/core-http";
+import { ServiceBusAtomXmlConstants } from "../util/constants";
+
+export class QueueResourceSerializer implements ResourceSerializer {
```

Comment:
Can we add the `@ignore` doc tag to anything we export that shouldn't be exposed to the user? This will ensure that our typedoc generated documentation doesn't display these classes/functions/fields.